api/client/webclient/webconfig.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 Gravitational, Inc.
+Copyright 2022 Gravitational, Inc.
```

Comment:
nit: I think we typically do this
```suggestion
Copyright 2015-2022 Gravitational, Inc.
```